receiver/sqlserverreceiver/scraper.go

```diff
@@ -504,19 +504,21 @@ func (s *sqlServerScraperHelper) recordDatabaseStatusMetrics(ctx context.Context
 
 func (s *sqlServerScraperHelper) recordDatabaseQueryTextAndPlan(ctx context.Context, topQueryCount uint) (plog.Logs, error) {
 	// Constants are the column names of the database status
-	const dbPrefix = "sqlserver."
-	const totalElapsedTime = "total_elapsed_time"
-	const rowsReturned = "total_rows"
-	const totalWorkerTime = "total_worker_time"
-	const queryHash = "query_hash"
-	const queryPlanHash = "query_plan_hash"
-	const logicalReads = "total_logical_reads"
-	const logicalWrites = "total_logical_writes"
-	const physicalReads = "total_physical_reads"
-	const executionCount = "execution_count"
-	const totalGrant = "total_grant_kb"
-	const queryText = "query_text"
-	const queryPlan = "query_plan"
+	const (
+		dbPrefix                   = "sqlserver."
+		executionCount             = "execution_count"
+		logicalReads               = "total_logical_reads"
+		logicalWrites              = "total_logical_writes"
+		physicalReads              = "total_physical_reads"
+		queryHash                  = "query_hash"
+		queryPlan                  = "query_plan"
+		queryPlanHash              = "query_plan_hash"
+		queryText                  = "query_text"
+		rowsReturned               = "total_rows"
+		totalElapsedTimeSecond     = "total_elapsed_time"
+		totalGrant                 = "total_grant_kb"
+		totalWorkerTimeMilliSecond = "total_worker_time"
```

Comment:
Could I know what is the meaning of this variable name `totalElapsedTimeSecond` and `totalWorkerTimeMilliSecond` while both `total_elapsed_time` and `total_worker_time` use microseconds in SQL server? Why do these two variables have different suffixes?

If you really want to add a suffix to the variable to show the original unit as the variable name, I suggest we use `microsecond` as the suffix. For instance, `totalWorkerTimeMicrosecond`. Then you can convert it to float64 and divide it by `1000000`  in `valueRetriever`.

This reduces the confusion by bringing fewer units into the code. No `millisecond`, only `microsecond` (original unit) and `second` (target unit)

Reply:
right, these`key` variable should not have suffix. i have removed them. thanks for pointing out.